### PR TITLE
Testing curriculum fixes

### PR DESCRIPTION
**Pull request by akanshmurthy**
@schaui6 @camillevilla 

There were minor hiccups (items in console boxes that didn't need to be typed out, naming of tests, bolding hints that were missed, directory paths, test clarity) when we did this curriculum at the recent RailsBridge in SF. This PR addresses the minor hiccups (not sure why it's picking up so many changes... it's just the first 3 files after the Gemfile that I changed).

### CODE REVIEW

**Comment by rachelmyers** [comment]
@akanshmurthy, let's get a PR that only has the changes you want. You could clean this one up, or the easiest way might be to create a new PR. If you go that route, make sure you stash or reset any other changes you may have locally, pull the newest master branch, and then copy over the changes you want. Let me know if you run into trouble.

**Comment by akanshmurthy** [comment]
@rachelmyers : not sure what you mean. This PR **does** have the changes we want. If you're concerned with the number of commits in this branch, you can squash and commit before you merge?

**Comment by rachelmyers** [comment]
@akanshmurthy Did you mean for all the changes in the `files changed` tab to be included in this PR? The whitespace changes to the Gemfile, jump out as something probably not intended. I meant let's get this to only include the changes you want to make, and not other changes.

**Comment by akanshmurthy** [comment]
@rachelmyers : sorry for the delay. I took a closer look at this and it seems like there's something wrong here. I believe that only the last commit (e40497f) is the one that needs to be merged as the rest of the commits are already in the master version of the code. I'll wait for @schaui6 @camillevilla to confirm before I move forward. 

It's also super hard to look at something that was written months ago and try to piece things together. It would be great if PRs could get reviewed in a timely fashion so that PR authors can remember what actually happened and new committers can be motivated to open a PR. When I've opened PRs in this repo, it's taken months (one time, I had to close a PR because it hadn't been reviewed in 7 months even after repeated bumps...).

**Comment by camillevilla** [comment]
@akanshmurthy thanks for the ping. I got caught up in the holiday rush this winter 😅 I'll have time to take a closer look this weekend and we can discuss what kind of rebase is in order.

**Comment by camillevilla** [comment]
hey @akanshmurthy, I did some rebasing and ironed out merge conflicts and pushed things up to [akanshmurthy/docs#rebase-check](https://github.com/akanshmurthy/docs/tree/rebase-check). Please confirm that these are the changes you wanted to make:
https://github.com/railsbridge/docs/compare/master...akanshmurthy:rebase-check

Once I get the thumbs up from you, I'll force push to [akanshmurthy/docs#testing-curriculum-fixes](https://github.com/akanshmurthy/docs/tree/testing-curriculum-fixes) and this PR will look nice n' neat for @rachelmyers review.

**Comment by akanshmurthy** [comment]
@camillevilla : no probs -> just to be clear, I was referring to a review from the maintainers of this repo not Sanderfer or you :)

In any case, yes. Your change is exactly what we need! Thanks so much!

**Comment by camillevilla** [comment]
Alrighty, force push is up @akanshmurthy! I'm always happy to provide a second set of eyes on testing curriculum stuff and make things easier for @rachelmyers to review.

**Comment by akanshmurthy** [comment]
Awesome, thanks a bunch! You rock!

File: sites/en/testing-rails-applications/types_of_tests.step [hunk 1]
```diff
@@ -109,14 +109,15 @@ steps do
 
     message "First, create a views folder in the spec folder. Then, create an oranges folder in the views folder. Lastly, create an oranges view spec file in the oranges folder. Type these commands in the terminal:"
 
-    console_without_message "mkdir app/spec/views"
-    console_without_message "mkdir app/spec/views/oranges"
-    console_without_message "cd app/spec/views/oranges"
+    console_without_message "mkdir spec/views"
+    console_without_message "mkdir spec/views/oranges"
+    console_without_message "cd spec/views/oranges"
```
Comment (rachelmyers) [inline sites/en/testing-rails-applications/types_of_tests.step]: These changes make total sense 👍

File: sites/en/testing-rails-applications/additional_concepts.step [hunk 2]
```diff
@@ -19,13 +19,14 @@ message <<-MARKDOWN
...
-  With spies, we are not talking about espionage... at least, not in relation to testing :) Spies can be used to verify whether a method was called on an object.
+  With spies, we are not talking about espionage... at least, not in relation to testing :) Spies can be used to verify whether a method was called on an object a certain number of times.
   For instance (assume you already have the orange double from above):
 
   ```ruby
   orange = spy('orange')
   orange.name
-  expect(orange).to have_received(:name)
+  orange.name
+  expect(orange).to receive(:name).exactly(2).times
```
Comment (rachelmyers) [inline sites/en/testing-rails-applications/additional_concepts.step]: Do you think it's worth adding the example of using `receive` on it's own and then showing this example of specifying the number of calls?

File: sites/en/testing-rails-applications/types_of_tests.step [hunk 3]
```diff
@@ -27,7 +27,7 @@ steps do
     source_code :ruby, <<-RUBY
       RSpec.describe Orange, :type => :model do
         context 'ActiveRecord associations' do
-          it 'Orange belongs to tree' do
+          it 'belongs to tree' do
```
[the remark this answers is not in the document]
Comment (akanshmurthy) [reply sites/en/testing-rails-applications/types_of_tests.step]: Cool, you're saying the change is good as is since Orange is removed from the it phrase?

**Comment by akanshmurthy** [comment]
@rachelmyers : thanks for the quick review! I was out on vacation this past week so I apologize for the delay on my end. But, I've addressed most if not all of your comments. Please take a look and let me know what you think; I should be pretty responsive as I'm back in the US.

**Comment by akanshmurthy** [comment]
@rachelmyers : pinging to bump this :) we have a RailsBridge workshop at the end of the month and would like to have this merged in for that workshop.

**Comment by rachelmyers** [comment]
Ah, thanks for the bump; this looks good!

**Comment by akanshmurthy** [comment]
Thanks!

**Pull request merged**